python/doc/examples/probabilistic_modeling/arma_manipulation.ipynb

```diff
@@ -0,0 +1,538 @@
+{
```

Comment:
Name the objects to get nice titles
The print of the realization is not usefull: it is better to draw it!

Reply:
It would be nice to have the nice expression of the ARMA created in the script with a latex formula......

Reply:
it is not useful to list at the beginning all the methods illustrated in the text.